pkg/crc/api/handlers.go

```diff
@@ -143,6 +143,11 @@ func (h *Handler) SetConfig(args json.RawMessage) string {
 		return encodeStructToJSON(setConfigResult)
 	}
 
+	if a["properties"] == nil {
+		setConfigResult.Error = "No config keys provided"
+		return encodeStructToJSON(setConfigResult)
+	}
+
 	configs := a["properties"].(map[string]interface{})
```

Comment:
You can also do: 

```
configs, ok := a["properties"].(map[string]interface{})
if !ok {
  setConfigResult.Error = "Invalid config"
  return encodeStructToJSON(setConfigResult)
}
``` 

It will throw an error if you pass a JSON array for instance.

Reply:
... while this is a better fix and prevents an error, this should also be prevented to be sent from the client(s)